### ISSUE TITLE
{} string interpolation fails for non-trivial cases with log4j

### ISSUE THREAD

**Issue by mighdoll** [typesafehub/scala-logging#88]
When we tried updating to 3.6.0, we found compile errors. 

Here's an example that works with 3.5.0 and fails with 3.6.0:

```scala
package example
import com.typesafe.scalalogging.StrictLogging

object Hello extends App with StrictLogging {
  val num = 1
  val str = "foo"
  val msg = s"hello $str $num $str"
  logger.info(msg)
  logger.info(s"hello $str $num")      // fails to compile
  logger.info(s"hello $str $num $str") // also fails
}
```

Here's a trivial project containing the above example: [example](https://github.com/mighdoll/scala-logging-bug)

**Comment by analytically** [comment]
Chang was introduced by https://github.com/typesafehub/scala-logging/commit/a49f0f3dbb5835d422f0e2f336d496a15fded43a - will investigate and release 3.7.0 asap.

**Comment by analytically** [comment]
Fixed in 3.7.0.
